modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -59,42 +59,64 @@ Preparing the bare metal node requires executing the following procedure from th
 apiVersion: v1
 kind: Secret
 metadata:
-  name: openshift-worker-<num>-bmc-secret
+  name: openshift-worker-<num>-bmc-secret <1>
 type: Opaque
 data:
-  username: <base64-of-uid>
-  password: <base64-of-pwd>
+  username: <base64-of-uid> <2>
+  password: <base64-of-pwd> <3>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: openshift-worker-<num>
+  name: openshift-worker-<num> <1>
 spec:
   online: true
-  bootMACAddress: <NIC1-mac-address>
+  bootMACAddress: <NIC1-mac-address> <4>
   bmc:
-    address: <protocol>://<bmc-ip>
-    credentialsName: openshift-worker-<num>-bmc-secret
-----
-+
-Replace `<num>` for the worker number of the bare metal node in the two `name` fields and the `credentialsName` field. Replace `<base64-of-uid>` with the `base64` string of the user name. Replace `<base64-of-pwd>` with the `base64` string of the password. Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC.
-+
-See the BMC addressing section for additional BMC configuration options. Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others.
-Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
+    address: <protocol>://<bmc-ip> <5>
+    credentialsName: openshift-worker-<num>-bmc-secret <1>
+  networkConfig: | <6>
+    interfaces:
+    - name: <NIC1_name>
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+        - ip: "<IP_address>"
+          prefix-length: 24
+        enabled: true
+    dns-resolver:
+      config:
+        server:
+        - <DNS_IP_address>
+    routes:
+      config:
+      - destination: 0.0.0.0/0
+        next-hop-address: <IP_address>
+        next-hop-interface: <NIC1_name>
+----
+<1> Replace `<num>` for the worker number of the bare metal node in the two `name` fields and the `credentialsName` field.
+<2> Replace `<base64-of-uid>` with the `base64` string of the user name.
+<3> Replace `<base64-of-pwd>` with the `base64` string of the password.
+<4> Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. See the BMC addressing section for additional BMC configuration options. Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others.
+<5> Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
+<6> Optional. You can set the `networkConfig` configuration option to set static IP addreses. Ensure that you pass the NMState configuration in as a YAML string, with the `|` pipe symbol indicating that it is a string. See "(Optional) Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for configuration details.
```

Comment:
@johnwilkins
As of 4.10.0-0.nightly-2022-01-21-231627 networkConfig no longer accepts string format (denoted by the pipe character)
it instead only accepts yaml format.
aside from removing the pipe character, behaviour is the same.